enterprise/frontend/src/embedding-sdk/components/public/StaticQuestion/StaticQuestion.tsx

```diff
@@ -109,6 +112,7 @@ const StaticQuestionInner = ({
         const { card, result } = await loadStaticQuestion({
```

Comment:
`loadStaticQuestion` is only used here, which makes me wonder if we should  make it into a hook so that we don't forget about the cleanup if we move the code around.
How do we solve this issue on the main app? Are we using redux toolkit there?

Anyway, I think we should add a test for this as it's easy to re-introduce the bug again when doing a refactor.
I'm wondering if it' possible to check if the request was aborted with jest, that should be a "refactor-proof" test

Reply:
Good idea, I'll write a test for the cleanup & see if I can extract `loadStaticQuestion` into a hook. I think the main app uses redux thunk to load the question, and reference it via the `getQuestion` selector.

Reply:
Fixed in https://github.com/metabase/metabase/pull/48808/commits/aee7d9c968ed6ee69f55a0b00023552d6c4ada69 and https://github.com/metabase/metabase/pull/48808/commits/4ed518b14ca7471f7140664dd08099ac0fd6645b